spec/models/gogovan_order_spec.rb

```diff
@@ -58,4 +58,8 @@
       expect(order.cancelled?).to eq(false)
     end
   end
+
+  describe 'callbacks' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.